patches/@oclif+dev-cli+1.22.0.patch

```diff
@@ -75,11 +75,17 @@ index 3e613e0..129d041 100644
 +        const sources = [
 +            'bin', 'build', 'patches', 'typings', 'CHANGELOG.md', 'INSTALL.md',
 +            'LICENSE', 'package.json', 'npm-shrinkwrap.json', 'README.md',
-+            'TROUBLESHOOTING.md',
++            'TROUBLESHOOTING.md', 'automation/check-npm-version.js',
```

Comment:
I've had to update this `oclif` patch because `oclif` copies the CLI source to a tmp folder to generate the native installers and runs `npm install` on that folder (as part of `npm run build:installer`), and now we have the `preinstall` hook that calls `check-npm-version.js`.